src/client.js

```diff
@@ -106,9 +100,11 @@ class NinaClient {
     } else if (/accounts\/(.*?)\/hubs/.test(url)) {
       const publicKey = url.split('/')[2];
       await this.processMulitpleHubAccountData(response.data.hubs)
-      for await (let hub of response.data.hubs) {
-        hub.accountData.collaborator = await this.processAndParseSingleHubCollaboratorAccountDataWithHub(publicKey, hub.publicKey)
-      }
+      await this.processMultipleHubCollaboratorAccountDataWithHubs(response.data.hubs, publicKey);
+      console.log('response: ', response.data)
+      // for await (let hub of response.data.hubs) {
```

Comment:
console + commented code should be removed even tho we dont have a linter in place yet